titanium-rdf-api-java/src/test/scala/eu/neveblink/jelly/convert/titanium/IntegrationSpec.scala

```diff
@@ -0,0 +1,93 @@
+package eu.neveblink.jelly.convert.titanium
+
+import com.apicatalog.rdf.api.RdfQuadConsumer
+import eu.neverblink.jelly.convert.titanium.TitaniumJellyReader
+import eu.neverblink.jelly.core.JellyOptions
+import eu.neverblink.jelly.core.proto.v1.*
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+import java.io.{ByteArrayInputStream, ByteArrayOutputStream}
+
+/**
+ * Integration tests for some Titanium-specific APIs.
+ */
+class IntegrationSpec extends AnyWordSpec, Matchers:
+  val testFrame = RdfStreamFrame.newInstance()
+    .addRows(
+      RdfStreamRow.newInstance()
+        .setOptions(JellyOptions.SMALL_STRICT.clone().setPhysicalType(PhysicalStreamType.QUADS))
+    )
+    .addRows(
+      RdfStreamRow.newInstance()
+        .setName(RdfNameEntry.newInstance().setValue("http://example.org/iri"))
+    )
+    .addRows(
+      RdfStreamRow.newInstance()
```

Comment:
Could you reuse the nicer `rdfQuad`-and-so-on API from `core-java` tests? To do this, you have to `.dependsOn(coreJava % "compile->compile;test->test)` on this module in build.sbt.